_pages/en/strategy-alternate-model.md

```diff
@@ -0,0 +1,360 @@
+3. Metrics to **measure the strategy's success** and compare it with the conventional delivery model
+
+The **business case** for this strategy is provided in [Appendix A](#appendix-a---business-case-diagnostic)
+<!-- markdownlint-disable MD001 -->
+##### Table of Content <!-- omit in toc -->
+<!-- markdownlint-enable MD001 -->
+- [Guiding Policy](#guiding-policy)
+- [Coherent set of actions](#coherent-set-of-actions)
+- [Measuring the Strategy's success](#measuring-the-strategys-success)
+- [Appendix A - Business Case (Diagnostic)](#appendix-a---business-case-diagnostic)
+- [Appendix B - Traceability Matrix](#appendix-b---traceability-matrix)
+- [Appendix C - References](#appendix-c---references)
+
+## Guiding Policy
+
+The following policy reflects the decision adopted by the **CIO of ESDC** when using this Alternative Solution Delivery Model.
+Each policy statement is a declaration of that decision and received the endorsement of its associated area of governance body (endorsements not yet obtained, see section coherent set of actions).
+
+***Governance and Compliance***
+
+1. IT Product teams have the authority to make changes to their IT products directly in production
+2. IT Product teams publish their product metrics internally to ESDC. Metrics comprises of the following:
+    - Accessibility test results
+    - General Quality test results
+    - Security test results
+    - API consumption statistics
+3. Security Assessment & Authority (SA&A) uses the _XXXAlternate-SA&AXXX methodology_
+4. Accessibility Assessment uses the _XXXAlternate-AccessibilityXXX methodology_
+5. Audit Assessment uses the _XXXAlternate-AuditXXX methodology_
+6. IT Projects do not use business requirements documents
+7. IT Projects start only when the following is true:
+    - Metrics of business outcomes are defined and approved by Business Architecture
+    - The business sponsor commits the availability of, at minimum, one business subject matter experts throughout the IT Project
+    - The business sponsor commits the availability of, at minimum, one policy expert throughout the IT Project
+    - The business sponsor commits the availability of a sample of end users throughout the IT Project
+
+***Finance***
+
+1. IT Projects are under $2.5M
+2. IT Projects are taxed $X% yearly; amount to be distributed amongst the IT products involved in the IT Project
+3. IT Projects do not use cost recovery codes, they fund IT Product cost centres
+4. IT Products use a showback billing model, not a cost recovery model
+
+***Architecture***
+
+1. IT Solutions use loose coupling architectures
+2. IT Products do not depend on other products to run in production
+3. IT Products can test, deploy, and make technical changes without dependencies on other product teams
+4. IT Product teams have the authority to choose their IT Products technical stacks
+5. IT Products expose their functionality and data via APIs
```

Comment:
We could even go further and say "All APIs should be written as though they are going to be publically available". Borrowed from the famous Jeff Bezos memo from ... some year. Cant find quickly on Google but early 2000s I think